spec/controllers/concerns/two_factor_authenticatable_methods_spec.rb

```diff
@@ -34,8 +34,17 @@
       result
     end
 
+    before do
```

Comment:
Can we not modify the default spec behavior and instead add this in a `context` block for the specific case where someone is authenticating after the timeframe has expired?